spec/javascripts/components/error-summary-spec.js

```diff
@@ -0,0 +1,72 @@
+/* global describe afterEach beforeEach it expect */
+
+var $ = window.jQuery
+
+function isFocused (element) {
+  return $(element)[0] === $(element)[0].ownerDocument.activeElement
+}
+
+describe('An error summary component', function () {
+  'use strict'
+
+  var GOVUK = window.GOVUK
+  var module
+  var element
+
+  beforeEach(function () {
+    element = $(
+      '<div ' +
+        'class="app-c-error-summary" ' +
```

Comment:
These tests should be failing

Reply:
Actually nevermind, it'd be nice to be consistent but they are just JavaScript so make sense to be passing.

Reply:
Everything other than the old `app-c` stuff here looks good :+1:

Reply:
Fixed.

The JavaScript tests weren't failing because the module is bound to the `js-error-summary__link` and isn't `app-` or `gem-` dependent.

We should use `gem-` here anyway for consistency.

Reply:
Good example of not binding to presentational classes 👍